easybuild/easyconfigs/d/Doxygen/Doxygen-1.8.14-GCCcore-7.2.0.eb

```diff
@@ -0,0 +1,19 @@
+name = 'Doxygen'
+version = '1.8.14'
+
+homepage = 'http://www.doxygen.org'
+description = """Doxygen is a documentation system for C++, C, Java, Objective-C, Python, 
+ IDL (Corba and Microsoft flavors), Fortran, VHDL, PHP, C#, and to some extent D."""
+
+toolchain = {'name': 'GCCcore', 'version': '7.2.0'}
+
+sources = ['%(namelower)s-%(version)s.src.tar.gz']
+source_urls = ['http://ftp.stack.nl/pub/users/dimitri/']
+
+builddependencies = [
+    ('flex', '2.6.4'),
```

Comment:
@rubendibattista You should include `binutils` `2.29` as a build dependency too when using `GCCcore` as toolchain (that's why the tests are failing).